test/class_decorator_test.ts

```diff
@@ -190,4 +190,37 @@ describe('Merging class decorators', () => {
       expect(baseDecoratorRegex.test(result.toString()), 'id of the patch should be present').to.be.false;
     });
   });
+});
+describe('Property without initialized value', () => {
+  const base = `  
+  @NgModule({
+    declarations: [AppComponent],
+    imports: [
+      StoreModule.forRoot(reducers, {
+        metaReducers,
+        runtimeChecks: {
+          strictStateImmutability: true,
+          strictActionImmutability: true,
+        },
+      }),
+    ],
+    providers: [],
+    bootstrap: [],
+  })
+  export class AppModule {}
+    `,
+    patch = ``;
```

Comment:
Actually here you are just testing the ability of the merger to map your `base` file but you are not testing the merging process.

You could add to `patch` a very similar code but changing one import name like:

```TypeScript
@NgModule({
    declarations: [AppComponent],
    imports: [
      StoreModule.forRoot(reducers, {
        blaBla,
        runtimeChecks: {
          strictStateImmutability: true,
          strictActionImmutability: true,
        },
      }),
    ],
    providers: [],
    bootstrap: [],
  })
```
Then please test that `blaBla` is added to the base.

Reply:
Done